springboard.yaml

```diff
@@ -43,7 +43,7 @@ models:
         type: string
 repositories:
```

Comment:
We should ideally have a `qa.springboard.yaml` and an alternate `prod.springboard.yaml` that is defined in puppet.